src/cms/cms.js

```diff
@@ -11,6 +11,7 @@ import AboutPreview from './preview-templates/aboutPreview'
 import SocialMediaPreview from './preview-templates/socialMediaPreview'
 import HeaderPreview from './preview-templates/headerPreview'
 import MemberSectionPreview from './preview-templates/memberSectionPreview'
+import SplashScreenPreview from './preview-templates/splashScreenPreview'
```

Comment:
@esayaskiros  This import statement is breaking all  the cms previews please make a PR to take it out

Reply:
done